app/env.server.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-var */
```

Comment:
we have declared a variable on the global object using `var`, since eslint throws an error when we declare `var ` 
 variables , i have disabled that warning

Reply:
env vars remix: https://remix.run/docs/en/1.14.0/guides/envvars
refer to the correct marked answeer: https://stackoverflow.com/questions/55030498/why-dont-const-and-let-statements-get-defined-on-the-window-object